x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/use_get_enriched_package_policies.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useEffect, useMemo, useState } from 'react';
+import { AgentPolicy, PackagePolicy } from '../../../../../../../common/types/models';
+import {
+  GetAgentPoliciesResponse,
+  GetAgentPoliciesResponseItem,
+} from '../../../../../../../common/types/rest_spec';
+import { useGetPackagePolicies } from '../../../../hooks/use_request';
+import {
+  SendConditionalRequestConfig,
+  useConditionalRequest,
+} from '../../../../hooks/use_request/use_request';
+import { agentPolicyRouteService } from '../../../../../../../common/services';
+import { AGENT_POLICY_SAVED_OBJECT_TYPE } from '../../../../../../../common/constants';
+import { GetPackagePoliciesResponse } from '../../../../../../../common/types/rest_spec';
+
+export interface PackagePolicyEnriched extends PackagePolicy {
+  _agentPolicy: GetAgentPoliciesResponseItem | undefined;
```

Comment:
is the naming convention with `_` just to denote that it's an enriched field and not a part of the saved objet?

Reply:
yes, exactly. I was looking for a property name that would not collide with any future addition to the object. I went with a prefix of `_` but had also thought about using a `$` suffix (ex. `agentPolicy$`)